en/docs/administer/managing-users-and-roles/managing-user-stores/configure-primary-user-store/configuring-a-jdbc-user-store.md

```diff
@@ -13,11 +22,18 @@ You can either use the default configuration or you can change it in the followi
 !!!tip
     Before you begin, ensure that the RDBMS that you want to use as the JDBC user store is correctly set up for your system. Then, follow the steps given below to configure a JDBC user store as the primary user store in your product.
 
+Follow the given steps to configure a JDBC user store:
+
+-   [Step 1: Configuring the JDBC user store manager](#step-1-configuring-the-jdbc-user-store-manager)
+-   [Step 2: Updating the system administrator](#step-2-updating-the-system-administrator)
+-   [Step 3: Updating the datasources](#step-3-updating-the-datasources)
+-   [Step 4: Starting the server](#step-4-starting-the-server)
+
 ### Step 1: Configuring the JDBC user store manager
 
-1. To configure a JDBC user store as the primary user store, you must give the value: `database` for the config: `user_store.type` in the `<APIM_HOME>/repository/conf/deployment.toml` file.
+-   Add these configurations below in `<API-M_HOME>/repository/conf/deployment.toml` file.
 
-    ```toml
+    ```
```

Comment:
Did we test configuring the userstore type as `database`. AFAIR, there were some issues when setting the userstore type as `database`. We need to set it as `database_unique_id` to resolve it.  could you please double check on this? :)

Reply:
Corrected https://github.com/wso2/docs-apim/pull/2174/commits/c21f3d0bd012d6cbf0be75733e110326ae5e53f3